packages/material-ui/src/Tabs/Tabs.js

```diff
@@ -338,10 +338,12 @@ const Tabs = React.forwardRef(function Tabs(props, ref) {
     };
   }, [updateIndicatorState, updateScrollButtonState]);
 
-  const handleTabsScroll = React.useCallback(
-    debounce(() => {
-      updateScrollButtonState();
-    }),
+  const handleTabsScroll = React.useMemo(
+    () =>
+      debounce(() => {
+        updateScrollButtonState();
+      }),
+    [updateScrollButtonState],
```

Comment:
This means we don't need to memoize. `updateScrollButtonState` always changes

Reply:
This dependency is also memoized: https://github.com/mui-org/material-ui/blob/9e2cbae16f08b166d5f97fb625348f4ac131b50d/packages/material-ui/src/Tabs/Tabs.js#L305.

Reply:
No idea what I looked at.

Reply:
useEventCallback will return the same reference between two renders which guarantee that we keep the same denounce method between two renders.

Reply:
I know. I remember seeing `const ... = () => {}`. Probably looked at the wrong line.